examples/worldview/src/features/stage/utils.js

```diff
@@ -0,0 +1,7 @@
+export const nearestEvenInt = num => 2 * Math.round(num / 2);
```

Comment:
what's the reason for diving num / 2 and then round and multiplying 2?

Reply:
The function needs to return an even number, in this example we can see how this works:
```
Math.round(13) = 13
2 * Math.round(13 / 2) = 14
```
[Any number multiplied by 2 is even.](https://www.theguardian.com/notesandqueries/query/0,5753,-2212,00.html#:~:text=EVEN%20NUMBERS%20can%20be%20looked,can%20be%20described%20as%202n.&text=Therefore%2C%20any%20even%20number%20plus,some%20number%20multiplied%20by%20two).)